source/common/network/connection_impl.cc

```diff
@@ -136,7 +136,7 @@ void ConnectionImpl::closeSocket(ConnectionEvent close_type) {
     return;
   }
 
-  ENVOY_CONN_LOG(debug, "closing socket: {}", *this, close_type);
+  ENVOY_CONN_LOG(debug, "closing socket: {}", *this, static_cast<uint32_t>(close_type));
```

Comment:
Might be worth a `<<` override, up to you. I wish C++ had a better stringification for `eum class` by default.